ver/us/undefined_syms.txt

```diff
@@ -67,6 +67,14 @@ D_7012BC11 = 0x7012BC11;
 D_80026AC7 = 0x80026AC7;
 D_8004AA85 = 0x8004AA85;
 
+// RSP Microcodes
+rspbootTextStart = 0xB0000B70;
+rspbootTextEnd = 0xB0000C70;
+n_aspMainTextStart = n_aspMain_text_bin;
+n_aspMainDataStart = n_aspMain_data_bin;
+gspF3DEX2_fifoTextStart = gspF3DEX2kawase_fifo_text_bin;
```

Comment:
I'm not sure these alias names that that important yet. I'd rather try to remove entries from this file than add..though I don't care too much